src/main/java/org/roaringbitmap/RunContainer.java

```diff
@@ -1506,6 +1506,10 @@ protected boolean isFull() {
     return (this.nbrruns == 1) && (this.getValue(0) == 0) && (this.getLength(0) == -1);
   }
 
+  public static Container full() {
+    return new RunContainer(1, new short[]{0, -1, 0, 0, 0, 0, 0, 0});
```

Comment:
Hi, my first try was just new short[]{0, -1} and it worked fine in all tests including new ones. As I'm not fully aware what reason was to have `DEFAULT_INIT_SIZE = 4` and new RunContainer().iadd(0, 1 << 16)` returns 6 trailing zeroes I decided to have it that way.